### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by Pandapip1**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-12345.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## Abstract
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.
+
+## Motivation
+The motivation section should describe the "why" of this EIP. What problem does it solve? Why should someone want to implement this standard? What benefit does it provide to the Ethereum ecosystem? What use cases does this EIP address?
+
+## Specification
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).
```
Comment (Pandapip1) [inline EIPS/eip-12345.md:22]:
```suggestion
The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms.
```
Shut up eipw

**Comment by Pandapip1** [comment]
:D

**Comment by Pandapip1** [comment]
@MicahZoltu would you mind approving this PR so I can E2E this?

**Review by SamWilsn — APPROVED** [review]

**Comment by Pandapip1** [comment]
Well it approved but it didn't automerge...

**Review by MicahZoltu — APPROVED** [review]

**Comment by Pandapip1** [comment]
That was stupid.

**Review by MicahZoltu — APPROVED** [review]

**Comment by Pandapip1** [comment]
:D

**Pull request merged**